### ISSUE TITLE
Images are not able to be uploaded as thumnail for notes

### ISSUE THREAD

**Issue by SidharthBansal** [publiclab/plots2#2189]
### Please describe the problem (or idea)
I tried to make a "research note" and upload an image as thumbnail but it is like this
![image](https://user-images.githubusercontent.com/20972099/35670416-8cdbc046-075e-11e8-9054-ebfb889e7d01.png)
After selecting publish it is not saving the image used for thumbnail of the post.

**Comment by jywarren** [comment]
Hi, is this on staging or on the production site?

Thanks! I recently had an issue with this on the new questions page...
wonder what's going on?

**Comment by Gauravano** [comment]
Hi @jywarren @SidharthBansal I have collected some logs regarding this issue which may help 

![erroruploadresearch](https://user-images.githubusercontent.com/20878070/35750024-96349804-0879-11e8-8d3e-00c34bf521da.png)
I like to point @jywarren that while editing in inline wiki editor same type of error is generated.

**Comment by SidharthBansal** [comment]
@Gauravano Thanks for the logs.
Actually I am having trouble with #2172 so was busy with fixing that. Sorry for not checking things and pending prs @jywarren 
@jywarren it is in both development and production with a slight difference. On production, we are not able to see the horizontal line with blue colors but on development, we are able to.
On both of them, it is not working.

**Comment by Zengirl2** [comment]
Just adding that I experienced the same problem using the Rich Text editor today in making a research note. It allows me to try to choose the image file, but it does not actually upload it, even though it is the correct size file. I tried this with two research notes and ended up using the Legacy editor instead (which worked fine). Also, just an additional note that on the Rich Text editor there is a "read more" on step 1 of creating the title for the research note that is not going to its link. It just returns you to the same page as if it doesn't have the correct link to go to.
